etc/cmake/options.cmake

```
@@ -49,6 +49,11 @@ if(OTBR_DBUS)
     target_compile_definitions(otbr-config INTERFACE OTBR_ENABLE_DBUS_SERVER=1)
 endif()
 
+option(OTBR_PROTO "Enable PROTO support" OFF)
```

It seems PROTO is a serializing standard. It seems too general to be used as a name for this feature.
```suggestion
option(OTBR_FEATURE_FLAGS "Enable feature flags support" OFF)
```

---

Done.